### ISSUE TITLE
Regarding tilemap and attrmap encoding

### ISSUE THREAD

**Issue by Rangi42** [zladx/LADX-Disassembly#344]
- `peach.tilemap` and `peach.attrmap` are committed, but none of the other decoded files are; should they be?
- Running `convert_background.py decode` on `menu_file_selection.tilemap.encoded` and `menu_file_selection.attrmap.encoded` fails with an `IndexError`. I think this is because they fallthrough into the subsequent `tilemap03` data. It might be more useful to combine those files, and instead of the `Tilemap03` and `Attrmap03` pointers, store either a raw address (like the `._07 dw $D651`) or an offset from the base pointer (e.g. `dw MenuFileSelectionAttrmap + $0123`).
- `Tilemap06` and `Tilemap09` might also belong combined with another file. This would explain why some decoded files fail to open in Tilemap Studio, e.g. `inventory.tilemap` complains "Attrmap is longer than tilemap."

**Comment by kemenaran** [comment]
Indeed, the decoded tilemaps and attrmaps are there by mistake, and should not have been included in the project. I opened #346 to remove them.

As for the combined/fallthrough tilemaps, merging them and storing them as an offset from a base pointer would definitely be a good idea, yes.
